index.html

```diff
@@ -3020,36 +2919,34 @@ <h2>Processing a Manifest</h2>
 						manifest</a>, when available.</li>
```

Comment:
(commenting here has this specific text wasn't affected by the PR):

> _base_: a URL string that represents the base URL for the manifest.

should link back to our definition of the manifest base URL.